test/parallel/test-vm-context-property-forwarding.js

```diff
@@ -32,3 +32,9 @@ assert.strictEqual(vm.runInContext('x;', ctx), 3);
 vm.runInContext('y = 4;', ctx);
 assert.strictEqual(sandbox.y, 4);
 assert.strictEqual(ctx.y, 4);
+
+const x = vm.createContext({ get a() { return 5; } });
+
+assert.strictEqual(x.a, 5);
```

Comment:
Does `IndexedPropertyGetterCallback` apply for properties returning index values or indexed properties?
```js
Reflect.defineProperty(x, '1', { get() { return 2 } })
```

Reply:
I just tested that scenario and in fact, `x[1] === 2`. If that's the question. I'm not sure I quite understand it.